fMRISurface/GenericfMRISurfaceProcessingPipeline.sh

```diff
@@ -56,8 +56,8 @@ SmoothingFWHM=`opts_GetOpt1 "--smoothingFWHM" $@`  # "${14}"
 GrayordinatesResolution=`opts_GetOpt1 "--grayordinatesres" $@`  # "${14}"
 RegName=`opts_GetOpt1 "--regname" $@`
 
-if [ "${RegName}" = "" ]; then
-    RegName="FS"
+if [ -z "${RegName}" ]; then
+    RegName="MSMSulc"
```

Comment:
Maybe I missed it, but I don't remember a request to change this default.  It may be a good change, it makes some sense with deprecating FS, but it does mean that the same settings and the new script will do something substantially different than before.  @glasserm @mharms?

Reply:
Good point...

Reply:
Since FS is deprecated, I assumed that MSMSulc should be the new default. Do I change back to FS?

Reply:
Matt indicated that he thinks the default should be changed.  I am okay with it changing.

Anyone who is surprised by this change, wanting the same processing as an earlier run, should really have just used the same older version of the pipelines as before, so it doesn't seem like it has any major downside.

Reply:
And if you were using FS, you probably didn't have MSM set up, so it is likely the new default won't run, and the user would be alerted to the change in that fashion.

Reply:
Yes the default for PostFreeSurfer and fMRISurface should be MSMSulc

Reply:
Changed in both PRs.